### DISCUSSION TITLE
Subscription deduplication

### DISCUSSION THREAD

**Discussion by valdoryu** [Akryum/vue-apollo#471] [Q&A]
Hi !
I'm using a query all over my app, so I made a mixin for it.
Thanks to query deduplication, one query only is launched, and all my app is updated.

Perfect.

I recently added a subscription to this query to update it when an admin make a change to the user.
However, for everytime I call the mixin, a websocket is opened (instead of being deduplicated).
Is this the expected behavior, or am I missing something ?

Thanks

**Comment by jpikora** [comment]
I'm experiencing the same issue, I guess it is more related to Apollo itself and I've asked similar question on new Apollo spectrum chat. I'll reply here if I get any news.

**Comment by jpikora** [comment]
https://github.com/apollographql/apollo-feature-requests/issues/74
